docs/en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -77,3 +77,24 @@ SilverStripe\Assets\Flysystem\FlysystemAssetStore:
 ```
 
 Note that this will not allow you to utilise certain file versioning features in 4.0.
+
+## Migrating substantial number of files
+
+The time it takes to run the file migration will depend on the number of files and their size. The generation of thumbnail will depend on the number and dimension of your images.
+
+If you are migrating a substantial number of files, you should run file migration task either as a queued job or on the command line. If the migration task fails or times out, you can start it again and will pick off where it left off.
+
+If your environement supports the _Imagick_ PHP library, you may want to use that library instead of _GD_. Imagick is considerably faster when resizing images. You can switch back to _GD_ after running the file migration task.
+
+[Changing the image manipulation driver to Imagick](/02_Developer_Guides/14_Files/02_Images#changing-the-manipulation-driver-to-imagick)
```

Comment:
Can you use `(images#changing-the-...)` instead of providing the full path here?